### PR TITLE
Add slsa v1 rc blog post

### LINKED ISSUE #606: SLSA v1.0: Release Candidate 1
This issue tracks the release candidate 1 (RC1) of the [SLSA v1.0 specification][project].

The bar for RC1 is that someone can read it and understand the most important concepts. Our goal is to get someone shipped ASAP to allow reviewers to start providing feedback while we polish the rest. It's OK for there to be minor inconsistencies or poor explanations that a motivated reader can get past. A blocker for RC1 would be something like a major missing concept or a major inconsistency, such as referring to code review  (present in v0.1 but deferred for v1.0).

Outstanding blockers:

* [x] #130
  * Includes subtasks
  * The actual conformance program is out of scope; just clarifying that there exists some means to establish trust in a builder.
* [x] Provenance v1.0 - verification section
* [ ] #592
  * Most reviewers will be familiar with v0.1, so it will significantly help their review if we summarize the differences.
* [x] #605
  * Blocking because we're claiming v1.0 is stable, yet we have a huge "not yet defined" section
* [x] Any other P0s from the [project] board?
* [x] Any other TODOs?
* [x] Create versioned directory "v1.0-rc1".

Deferred:

* ~~Maybe additional work on verifying artifacts w.r.t. package ecosystems?~~
* ~~#567~~
* ~~#580~~

[project]: https://github.com/orgs/slsa-framework/projects/1/views/1

### PR DESCRIPTION
**Pull request by olivekl**
Add slsa.dev blog post announcing v1.0 release candidate.

Fixes #606.

### CODE REVIEW

File: docs/_posts/2023-02-24-slsa-v1-rc.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+A significant conceptual change is the division of SLSA's level requirements
+into multiple tracks. Previously, each SLSA level encompassed requirements
+across multiple software supply chain aspects: there were source, build,
+provenance, and common requirements. To reach a particular level, adopters
+needed to meet
+[all requirements in each of the four areas](/spec/v0.1/requirements).
+Organizing the specification in that way made adoption cumbersome, since
+requirements were split across unrelated domains—improvements in one area were
+not recognized until improvements were made in all areas.
```
Comment (hayleycd) [inline docs/_posts/2023-02-24-slsa-v1-rc.md:41]:
```suggestion
not recognized until improvements were made in all areas.
```

**Review by kpk47 — APPROVED** [review]

**Comment by MarkLodato** [comment]
I plan to send out a PR to create a "v1.0-rc1" directory with a snapshot of the spec so that it's not changing out from under reviewers. Let's wait to submit this until that's done.

**Review by di — APPROVED** [review]

**Review by MarkLodato — APPROVED** [review]
Thanks!

**Review by trishankatdatadog — APPROVED** [review]

**Comment by MarkLodato** [comment]
Submitting now. Thanks everyone for contributing!

**Pull request merged**